zebrad/tests/acceptance.rs

```diff
@@ -215,6 +236,10 @@ fn version_no_args() -> Result<()> {
 
     output.stdout_contains(r"zebrad [0-9].[0-9].[0-9]")?;
 
+    // Make sure we have no info message in output
+    let notfound = output.stdout_contains(r"INFO");
+    assert!(notfound.is_err());
+
```

Comment:
Let's delete the log message check, and make sure the output only contains the version.

You could write a `stdout_matches` function, like `stdout_contains`, but the whole output has to match.

Let's also reject any extra leading or trailing output:
```suggestion
    output.stdout_contains(r"^zebrad [0-9].[0-9].[0-9]$")?;
```